packages/app/src/cli/services/deploy.ts

```diff
@@ -52,6 +52,18 @@ export const deploy = async (options: DeployOptions) => {
       }
     }),
   )
+  if (environment.local.useThemeBundling()) {
+    const themeExtensions = await Promise.all(
+      options.app.extensions.theme.map(async (extension) => {
+        return {
+          uuid: identifiers.extensions[extension.localIdentifier]!,
+          config: '{"theme_extension": {"files": {}}}',
```

Comment:
is this hardcoded config correct?

Reply:
Code within core will populate the `files` field.